generator/metaline_builder.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
If you declare a class MetalinesBuilder, please, be so kind to put it in the file named metalines_builder. If you wish to keep the file named metaline_builder, please, rename the class.